doc/user/metrics/service-monitor.md

```diff
@@ -0,0 +1,45 @@
+## Using the ServiceMonitor prometheus-operator CRD
```

Comment:
Nit: Rephrasing the title for clarity.
```suggestion
## Using the ServiceMonitor to access metrics with Prometheus
```

Reply:
Not sure I agree with this, as ServiceMonitor is only a concept within prometheus-operator, I would include the name of it in the title.

Reply:
Fair point. I think I've gotten used to associating it directly with Prometheus since the operator is always involved.